examples/function/flatmap_stream/README.md

```diff
@@ -0,0 +1,10 @@
+# Flatmap Stream Example
+
+## Build image and push
+
+```shell
+make image
+# Privilege needed.
+docker push quay.io/numaio/numaflow-python/map-flatmap-stream:latest
+```
+
```

Comment:
can we use a multi-platform image build command?

Reply:
Not sure which command exactly you are referring to? `buildx`?